app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    friend_num = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
```

Comment:
The `except VaccineError` block will catch both `NotVaccinatedError` and `OutdatedVaccineError` since they inherit from `VaccineError`. If you want to handle these errors separately, you should catch them individually. Otherwise, ensure that the message 'All friends should be vaccinated' is appropriate for both cases.